docs/guide/README.md

```diff
@@ -30,6 +30,60 @@ yo open-wc
 - [Publishing](/publishing/)
 - [Automating](/automating/)
 
+## Why `open-wc`
+
+We want to provide the community with well-known and experience-tested recommendations for their web component projects. Everything we do here is to encourage and support development with these open standards.
+
+### Our Philosophy
+
+The goal of Open Web Components is to empower developers with powerful and battle-tested tools for creating and sharing open source web components.
+
+Many web developers have experienced the dreaded "Javascript Fatigue". With our recommendations, we hope you'll enjoy the peace of mind that comes from having a well-known default solution for almost everything. From IDE to CI, `open-wc` has got you covered.
+
+We want web component development to be accessible and approachable for people, so our recommendations aim to:
+- Be easy to use
+- Be ready to use
+- Give you that 'it just works' feeling
+
+### Why Web Components
+
+We believe that the open web is an ideal platform for application development. It's free, open, and standardized nature reenforces user's privacy and control while enabling developers to produce useful apps.
+
+#### Historical Background
+
+**tl;dr:** Libraries provided the inspiration to standardize the web's component model, which became a practical reality in 2019 after many years of work.
```

Comment:
```suggestion```

Reply:
The motivation for putting this here was because the history section is a little long (although I did revise a few times to make it as concise as possible)

Reply:
I think the history section is short enough to not have to include a tl;dr, I also feel like a tl;dr is more appropriate in e.g. a blog post instead of here

Reply:
what do you think of splitting this page into `our philosophy` and `why web components`?

Reply:
I think there will be more sections => I'm still missing our rationals for what we recommend (or our requirements that need to be met before we recommend?)